Slim/Menu/AlbumInfo.pm

```diff
@@ -273,7 +273,6 @@ sub infoContributors {
 		$linkRoles{'TRACKARTIST'} = 1;
 		$linkRoles{'ALBUMARTIST'} = 1;
 
-
```

Comment:
Please try to avoid no-op changes. Reset this file and get rid of it from the change set.